### PR TITLE
fixed hamburger bug

### PR DESCRIPTION
**Pull request by PeculiarGoat**
No more error in console.  X icon works like the one on Lonely Planet.

### CODE REVIEW

File: site/templates/nav.jet [hunk 1]
```diff
@@ -13,7 +13,7 @@
         </form>
       {{end}}
       
-      <button class="navbar-toggler" type="button" s72-collapse=".navbar-collapse" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+      <button class="navbar-toggler" type="button" toggle-function="toggleMobileMenu" s72-collapse=".navbar-collapse" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (gscragg) [inline site/templates/nav.jet:16]: Where does this new attribute get used?

Comment (PeculiarGoat) [reply site/templates/nav.jet:16]: [Here](https://github.com/shift72/core-template/blob/51d1dd5ba0ad9d508d944b75608d96cc1d3c4451/site/static/js/main.js#L407).

File: site/static/js/main.js [hunk 2]
```diff
@@ -421,17 +421,9 @@ function initializeMenu() {
...
-      if(icon) icon.classList.remove('toggled');
-      element.classList.remove('visible');
-    }
-    else
-    {
-      element.classList.add('visible');
-      if(icon) icon.classList.add('toggled');
-    }
+  var element = document.querySelector('.collapse.navbar-collapse');
+  if (element && icon) {
```
Comment (stajs) [inline site/static/js/main.js:425]: This is nicer from the "less lines of code" POV, but is it possible to not have an icon? The previous more verbose code did cater for that situation.

Comment (PeculiarGoat) [reply site/static/js/main.js:425]: It didn't... it checked for icon in the if and the else.

Comment (stajs) [reply site/static/js/main.js:425]: Yeah, my poor phrasing - let me rephrase. Before, it still added or removed a class on the element (and didn't matter if there was an icon), is that adding/removing class on the element no longer required (or handled elsewhere)?

Comment (PeculiarGoat) [reply site/static/js/main.js:425]: Ah right... yeah it's no longer required.  This js file was copy/pasted (I assume) from elsewhere but the template containing the elements that it selects wasn't so the `show` class is used instead of the `visible` class and seems to be handled elsewhere.
